backend/migrator/migration/prod/2.18/0002##user_group.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE user_group (
+    id SERIAL PRIMARY KEY,
+    row_status row_status NOT NULL DEFAULT 'NORMAL',
+    creator_id INTEGER NOT NULL REFERENCES principal (id),
+    created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    updater_id INTEGER NOT NULL REFERENCES principal (id),
+    updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    name TEXT NOT NULL,
+    resource_id TEXT NOT NULL,
```

Comment:
I was thinking of using email as the resource_id.

Reply:
LGTM, updated